lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -72,7 +72,7 @@ public IMRUJobDefinitionBuilder()
             _updateTaskMemory = 512;
             _coresPerMapper = 1;
             _updateTaskCores = 1;
-            _maxRetryNumberInRecovery = 0;
+            _maxRetryNumberInRecovery = -1;
```

Comment:
User can set 0, meaning they don't want to do any retry, or any number greater than 0. -1 is to indicate it is not set by user. Of course user can also set -1, but that doesn't make any sense.